app/helpers/template_helpers.py

```diff
@@ -155,6 +154,7 @@ def survey_config_mapping(*, theme: str, language: str, base_url: str) -> Survey
         "northernireland": SurveyConfig,
         "census": (WelshCensusSurveyConfig if language == "cy" else CensusSurveyConfig),
         "census-nisra": CensusNISRASurveyConfig,
+        "ni-business": NorthernIrelandBusinessSurveyConfig,
```

Comment:
This config should be under the current `northernireland` theme. I prefer what you have but requires changes on author end as well, so that is something we can look to rationalise in future.

Reply:
This also will mean we have a test_schema with this on

Reply:
Ya